File: Demo2D/Assets/Scripts/Globals/HomeControlCenter.cs
using UnityEngine;
using System.Collections;

public class HomeControlCenter : MonoBehaviour {

	private GameObject guimanager;

	// Use this for initialization
	void Start () {
		guimanager = GameObject.Find ("guiManager");
		GameControl gc = (GameControl)guimanager.GetComponent<GameControl>();
		gc.DrawHomeMainMenu();

	}
}

File: Demo2D/Assets/Scripts/Globals/HomeControlCenter.js
#pragma strict

var guimanager : GameObject;

// Use this for initialization
function Start () {
	guimanager = GameObject.Find ("guiManager");
	var gc : GameControl = guimanager.GetComponent(GameControl); //guimanager.GetComponent('GameControl');
	gc.DrawHomeMainMenu();
	gc.SetPlaying(false);
}